bt_executor: #do not change or remove the parameter namespace
  
  #Example of skills configurations
  comau_move_test:
    group_name: "comau"
    ik_service_name: "/comau_ik_solver/get_ik"
    location_name: "test_comau_location"
    fjt_action_name: "/comau_scaled_fjt_controller/follow_joint_trajectory"
    simulation: False
    speed_scaling_topic: "/speed_ovr"
    scaling: 50
    pipeline_id: "ompl"
    planner_id: "BiTRRT"
    velocity_scaling_factor: 1
    acceleration_scaling_factor: 1
    joints_weights: [1,1,1,1,1,1]

  comau_test_trj:
    group_name: "comau"
    trj_names: ["trj_comau"]
    fjt_action_name: "/comau_scaled_fjt_controller/follow_joint_trajectory"
    speed_scaling_topic: "/speed_ovr"
    scaling: 50
    repetitions: 5
    recompute_time_law: False
    simulation: False

  kuka_test_trj:
    group_name: "kuka"
    trj_names: ["trj_kuka"]
    fjt_action_name: "/kuka_scaled_fjt_controller/follow_joint_trajectory"
    repetitions: 2
    recompute_time_law: False
    simulation: False
    speed_scaling_topic: "/speed_ovr"
    scaling: 50

  trigger_service_test:
    pause: 1.5 #seconds of pause before triggering

  set_bool_service_test:
    pause: 1.5 #seconds of pause before setting the value
    value: True
  
  gpio_sequence_write_service_test:
    name: ["io1","io2"] #names of the input to write
    pause: [100,500] #pause in ms after inputs activations.
    value: [True, False] #value to set

  wait_test:
    wait_ms: 1000 #ms
